### PR TITLE
Find inputs in command  for RUN_ALL_REMOTELY

### PR DESCRIPTION
**Pull request by ramymedhat**

### CODE REVIEW

**Comment by ramymedhat** [comment]
@ola-rozenfeld  @gkousik

**Comment by gkousik** [comment]
LGTM

File: src/main/cc/proxy_client/proxy_client.cc [hunk 1]
```diff
@@ -141,6 +141,61 @@ string GetCompilerDir(const char* compiler) {
   return compiler_dir;
 }
 
+string Trim(const string& cmd) {
```
Comment (ola-rozenfeld) [inline src/main/cc/proxy_client/proxy_client.cc:144]: absl has a StripWhitespace function.

File: src/main/cc/proxy_client/proxy_client.cc [hunk 2]
```diff
@@ -328,6 +383,9 @@ int ComputeInputs(int argc, char** argv, const char** env, const string& cwd, co
     inputs->insert("prebuilts/jdk/jdk9/linux-x86");
     inputs->insert("external/icu");
     FindAllFilesFromCommand(argc, argv, inputs);
+  } else if (getenv("RUN_ALL_REMOTELY")) {
+    use_args_inputs = true;
+    FindAllFilesFromCommand(argc, argv, inputs);
```
Comment (ola-rozenfeld) [inline src/main/cc/proxy_client/proxy_client.cc:409]: We really need to clean up this entire function. Not in this CL, but in general. Lines 312-330 already try to figure out the inputs from command, but badly, and now you're doing it here better. We need to consolidate these.

File: src/main/cc/proxy_client/proxy_client.cc (hunk 1 above)
Comment (gkousik) [reply src/main/cc/proxy_client/proxy_client.cc:144]: This trims any non-alphabets in the beginning and end of string - a little different from abls's StripWhitespace

**Review by ola-rozenfeld — APPROVED** [review]

File: src/main/cc/proxy_client/proxy_client.cc (hunk 2 above)
Comment (ramymedhat) [reply src/main/cc/proxy_client/proxy_client.cc:409]: +1. Added to backlog (b/131301437).

**Comment by ola-rozenfeld** [comment]
Also, why does it show all the changes from the previous commits here? You rebased your changes on top of my changes, no?

**Comment by ramymedhat** [comment]
I see include_stats in the help string, line 704?
Yes just rebased on top of your changes before sending my last commit.

**Review by ola-rozenfeld — APPROVED** [review]

**Comment by ramymedhat** [comment]
Added include_stats to the help string.

**Pull request merged**